zqd/core.go

```diff
@@ -5,26 +5,29 @@ import (
 	"sync/atomic"
 
 	"github.com/brimsec/zq/pkg/iosrc"
+	"github.com/brimsec/zq/zqd/process"
 	"github.com/brimsec/zq/zqd/space"
-	"github.com/brimsec/zq/zqd/zeek"
 	"go.uber.org/zap"
 )
 
+type Launchers struct {
+	Suricata, Zeek process.Launcher
+}
```

Comment:
Rather than create a Launchers struct why not just add these as separate values in the Config struct?